### DISCUSSION TITLE
3.5.1

### DISCUSSION THREAD

**Discussion by slorber** [facebook/docusaurus#10386] [Release feedback]
#### :bug: Bug Fix

- `docusaurus-plugin-content-blog`, `docusaurus-theme-search-algolia`
  - [#10384](https://github.com/facebook/docusaurus/pull/10384) fix(core): algolia context import ([@slorber](https://github.com/slorber))
- `docusaurus-theme-search-algolia`
  - [#10382](https://github.com/facebook/docusaurus/pull/10382) fix(theme-algolia): useDocusaurusContext import error ([@anaclumos](https://github.com/anaclumos))

#### Committers: 2

- Sunghyun Cho ([@anaclumos](https://github.com/anaclumos))
- Sébastien Lorber ([@slorber](https://github.com/slorber))

<hr /><em>This discussion was created from the release <a href='https://github.com/facebook/docusaurus/releases/tag/v3.5.1'>3.5.1</a>.</em>

**Comment by amanaknows** [comment]
<a href="https://github.com/iAngelica/dark-side-repo" target="_blank">
    <img src="https://img.shields.io/badge/Sith%20Lord%20S5-AI%20Automation%20Dark%20Side%20Control-black?style=for-the-badge&logo=dark-mode&logoColor=white&labelColor=black" alt="AI Automation Dark Side Control Badge">
</a>
